pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtl.java

```diff
@@ -187,7 +187,7 @@ static void runFhirJdbcFetch(FhirEtlOptions options, DatabaseConfiguration dbCon
 		EtlUtils.logMetrics(result.metrics());
 	}
 	
-	static void validateOptions(FhirEtlOptions options) {
+	static void validateOptionsAndInit(FhirEtlOptions options) throws SQLException, PropertyVetoException {
```

Comment:
What do you think about having a separate init function for the DB? To me, it doesnt seem there is a relation between validating the option and initialising, especially when the initialising only happens when `getSinkDbUrl` is not empty?

Can we move the initialisation to the `setup` function in `FetchSearchPageFn` instead?